lumigator/frontend/package.json

```diff
@@ -14,7 +14,16 @@
     "type-check": "vue-tsc --build",
     "lint": "eslint . --fix",
     "lint:fix": "eslint . --fix",
-    "format": "prettier --write src/ e2e/"
+    "format": "prettier --write src/ e2e/",
+    "prepare": "cd ../../ && husky install lumigator/frontend/.husky"
+  },
+  "lint-staged": {
+    "*.{js,jsx,ts,tsx,vue}": [
+      "eslint --fix"
+    ],
+    "*.{json,js,ts,jsx,tsx,vue,html,css,scss}": [
+      "prettier --write"
+    ]
   },
```

Comment:
this configured lint-staged to call these commands whenever any of the files on the left side changes so it doesn't run against the whole project, but only the staged files matching this config.